Transforms/CachedQvvs/Editor/QvvsSceneViewFocus.cs

```diff
@@ -0,0 +1,41 @@
+using Latios.Transforms;
+using Unity.Entities.Editor;
+using Unity.Rendering;
+using UnityEditor;
+using UnityEngine;
+
+static class QvvsSceneViewFocus
```

Comment:
This needs to also work in Unity Transforms mode. And should probably be general purpose for that.

I think this editor script should be moved to Kinemation's Editor and this type should be renamed to `RuntimeEntitySceneViewFocus`.